src/display.jl

```diff
@@ -16,11 +17,11 @@ function yticklabels(tfr::MFCC, yticks::Array)
     map(Int, map(round, yticks)), "MFCC number"
 end
 
-heatmap(tfr::DSP.Periodograms.TFR) = log10(power(tfr))
+heatmap(tfr::DSP.Periodograms.TFR) = map(x -> log(10,x), tfr.power)
```

Comment:
```suggestion
heatmap(tfr::DSP.Periodograms.TFR) = log10.(power(tfr))
```
I cannot see why use `tfr.power` instead of `power(tfr)`, I think the latter (original) is more flexible